activationdemo/source/java/src/org/kaaproject/kaa/demo/activation/KaaManager.java

```diff
@@ -0,0 +1,131 @@
+
+/**
+ * Class that shows Kaa activation API, works with endpoints and it's profiles
+ *
+ * @see <a href="http://docs.kaaproject.org/display/KAA/Endpoint+profiling">Profiling API</a>
+ */
+public class KaaManager {
+
+    public static final int KAA_CLIENT_NUMBER = 9;
+    public static final String PROPERTIES_OUT_DIR = "res/out/";
+    public static final String KAA_PROPERTIES_DIR_PREFIX = "kaaTempDir";
+
+    private static final Logger LOG = LoggerFactory.getLogger(KaaManager.class);
+
+    private ArrayList<KaaClient> kaaClients = new ArrayList<>();
+
+    public void startKaaClient(final int index) throws IOException {
+
+        KaaClientProperties kaaClientProperties = new KaaClientProperties();
+        kaaClientProperties.setWorkingDirectory(KaaManager.PROPERTIES_OUT_DIR + KaaManager.KAA_PROPERTIES_DIR_PREFIX + index);
+
+        // Create the Kaa desktop context for the application.
+        DesktopKaaPlatformContext desktopKaaPlatformContext = new DesktopKaaPlatformContext(kaaClientProperties);
+
+        // Create a Kaa client and add a listener which displays the Kaa client activation
+        // as soon as the Kaa client is started.
+        final KaaClient kaaClient = Kaa.newClient(desktopKaaPlatformContext, new SimpleKaaClientStateListener() {
+            @Override
+            public void onStarted() {
+                super.onStarted();
+
+                displayConfiguration(index);
+            }
+        });
+        kaaClients.add(kaaClient);
+
+        kaaClient.setProfileContainer(new ProfileContainer() {
+            @Override
+            public PagerClientProfile getProfile() {
+                PagerProfileData.InnerData data = PagerProfileData.init().get(index);
+
+                PagerClientProfile pagerClientProfile = new PagerClientProfile();
+                pagerClientProfile.setAudioSupport(data.isAudio());
+                pagerClientProfile.setVibroSupport(data.isVibro());
+                pagerClientProfile.setVideoSupport(data.isVideo());
+
+                return pagerClientProfile;
+            }
+        });
+
+        // Persist activation in a local storage to avoid downloading it each time the Kaa client is started.
+        kaaClient.setConfigurationStorage(new SimpleConfigurationStorage(desktopKaaPlatformContext, "saved_config.cfg"));
+
+        // Add a listener which displays the Kaa client activation each time it is updated.
+        kaaClient.addConfigurationListener(new ConfigurationListener() {
+            @Override
+            public void onConfigurationUpdate(PagerConfiguration configuration) {
+                displayConfiguration(index);
+            }
+        });
+
+        // Start the Kaa client and connect it to the Kaa server.
+        kaaClient.start();
+
+    }
+
+    public void stopKaaClients() throws IOException {
+        LOG.info("Stopping clients...");
+        for (KaaClient k : kaaClients) {
+            if (k != null)
+                k.stop();
```

Comment:
add curly braces to if statement please, try to follow [google code style](https://google.github.io/styleguide/javaguide.html#s4.1-braces)